### PR TITLE
LINE web login Version 2.1

### PR DESCRIPTION
**Pull request by KosukeKamiya**

### CODE REVIEW

**Comment by YichingChiang** [comment]
id_tokenからprofile情報を取得する仕様は次回アップデートに入れることでしょうか？

**Comment by YichingChiang** [comment]
looks like this PR didn't include obtaining the profile information from id_token. Will it be added to next update? anyway, I think it's fine.

**Comment by KosukeKamiya** [comment]
@YichingChiang @mrmaakun 
Yes. 
I think obtaining the profile information from id_token should be added in the next update.

File: src/main/java/com/linecorp/sample/login/infra/line/api/v2/LineAPIService.java [hunk 1]
```diff
@@ -63,33 +63,39 @@ public AccessToken refreshToken(final AccessToken accessToken) {
...
             throw new RuntimeException(e);
         }
 
-        return "https://access.line.me/dialog/oauth/weblogin?response_type=code"
+        return "https://access.line.me/oauth2/v2.1/authorize?response_type=code"
                 + "&client_id=" + channelId
                 + "&redirect_uri=" + encodedCallbackUrl
-                + "&state=" + state;
+                + "&state=" + state
+                + "&scope=openid%20profile"
```
Comment (cosmic-cowboy) [inline src/main/java/com/linecorp/sample/login/infra/line/api/v2/LineAPIService.java]: @KosukeKamiya 
scope parameter value is only "openid%20profile" ?
Does scope parameter have other value?

**Pull request merged**